protocols/data_processor.py

```
@@ -4,101 +4,105 @@
     """Process data files and extract insights"""
     # Try multiple possible data directories
     possible_dirs = [
-        os.environ.get('DATA_DIR', '/data'),
-        '/data',
-        '/app/data',
-        '/tmp',
-        os.getcwd()
+        os.environ.get("DATA_DIR", "/data"),
+        "/data",
+        "/app/data",
+        "/tmp",
```

Ruff (0.11.9)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)